### ISSUE TITLE
Link to the data

### ISSUE THREAD

**Issue by tevig** [SamAmco/track-and-graph#178]
there is a hierarchy in the application ('folders' and 'files'). it was very convenient to create a 'symbolic link to files'. For example, a event = warm-up (10 minutes) - could lie:
 - /Root/Morning/Warm-up; 
 - /Root/Sports/Warm-up

**Comment by SamAmco** [comment]
Interesting. Definitely sounds like a good idea. I'll consider it at some point but I won't likely get to it for a good while. I assume you mean to track the same thing from multiple locations?

**Comment by tevig** [comment]
Yes, to be able to add data to the same track from different locations

**Comment by SamAmco** [comment]
I finally did this
